Build errors and warnings due to recent property name changes

With the recent property and class name changes to Manager ([here](https://github.com/apache/accumulo/pull/1904) I think) builds of accumulo-testing now fail. There are also a few warnings that relate to other recent changes.

<details>
<pre>
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/Module.java:[46,37] cannot find symbol
  symbol:   class SimpleThreadPool
  location: package org.apache.accumulo.core.util
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Replication.java:[20,1] cannot find symbol
  symbol:   static MASTER_REPLICATION_SCAN_INTERVAL
  location: enum org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Setup.java:[29,37] cannot find symbol
  symbol:   class SimpleThreadPool
  location: package org.apache.accumulo.core.util
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/Module.java:[227,35] cannot find symbol
  symbol:   class SimpleThreadPool
  location: class org.apache.accumulo.testing.randomwalk.Module
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Replication.java:[77,22] cannot find symbol
  symbol:   variable MASTER_REPLICATION_SCAN_INTERVAL
  location: class org.apache.accumulo.testing.randomwalk.concurrent.Replication
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[87,35] cannot find symbol
  symbol:   variable MASTER_BULK_THREADPOOL_SIZE
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[88,35] cannot find symbol
  symbol:   variable MASTER_BULK_RETRIES
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[89,35] cannot find symbol
  symbol:   variable MASTER_BULK_TIMEOUT
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[90,35] cannot find symbol
  symbol:   variable MASTER_FATE_THREADPOOL_SIZE
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[91,35] cannot find symbol
  symbol:   variable MASTER_RECOVERY_DELAY
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[92,35] cannot find symbol
  symbol:   variable MASTER_LEASE_RECOVERY_WAITING_PERIOD
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[93,35] cannot find symbol
  symbol:   variable MASTER_THREADCHECK
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java:[94,35] cannot find symbol
  symbol:   variable MASTER_MINTHREADS
  location: class org.apache.accumulo.core.conf.Property
[ERROR] /home/jeffreymanno/git/accumulo-testing/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Setup.java:[64,32] cannot find symbol
  symbol:   class SimpleThreadPool
  location: class org.apache.accumulo.testing.randomwalk.bulk.Setup
[INFO] 14 errors 

</pre>
</details>

<details>
<pre>
WARNING] COMPILATION WARNING : 
[INFO] -------------------------------------------------------------
[WARNING] Cannot find annotation method 'since()' in type 'java.lang.Deprecated'
[WARNING] Cannot find annotation method 'since()' in type 'java.lang.Deprecated'
[WARNING] Cannot find annotation method 'since()' in type 'java.lang.Deprecated'
[WARNING] Cannot find annotation method 'since()' in type 'java.lang.Deprecated'
[WARNING] Cannot find annotation method 'since()' in type 'java.lang.Deprecated'

</pre>
</details>

<details>
<pre>
[WARNING] Invalid project model for artifact [commons-vfs2:org.apache.commons:2.6.0]. It will be ignored by the remote resources Mojo.
[WARNING] Invalid project model for artifact [accumulo-hadoop-mapreduce:org.apache.accumulo:2.1.0-SNAPSHOT]. It will be ignored by the remote resources Mojo.
[WARNING] Invalid project model for artifact [accumulo-core:org.apache.accumulo:2.1.0-SNAPSHOT]. It will be ignored by the remote resources Mojo.
[WARNING] Invalid project model for artifact [accumulo-start:org.apache.accumulo:2.1.0-SNAPSHOT]. It will be ignored by the remote resources Mojo.

</pre>
</details>

**Versions:**
 - Affected version(s) of this project: recent changes of  2.1.0-SNAPSHOT
 
 **To Reproduce**

1.  Use fluo-uno to start up a cluster with the most recent changes in 2.1.0-snapshot
2.  Configure accumulo-testing to use that cluster
3.  rebuild with maven if necessary and then run the build script in `accumulo-testing/bin`

--- a/contrib/import-control.xml
+++ b/contrib/import-control.xml
@@ -35,13 +35,13 @@
 
     <!-- exceptions for testing -->
     <allow pkg="org.apache.accumulo.core.conf"/>
+    <allow class="org.apache.accumulo.core.util.threads.ThreadPools"/>
 
     <!-- TODO refactor code to remove the following exceptions -->
     <allow class="org.apache.accumulo.core.metadata.MetadataTable"/>
     <allow class="org.apache.accumulo.core.replication.ReplicationTable"/>
     <allow class="org.apache.accumulo.core.spi.scan.HintScanPrioritizer"/>
     <allow class="org.apache.accumulo.core.clientImpl.TabletServerBatchWriter"/>
-    <allow class="org.apache.accumulo.core.util.SimpleThreadPool"/>
     <allow class="org.apache.accumulo.core.util.FastFormat"/>
     <allow class="org.apache.accumulo.core.util.Pair"/>
     <allow class="org.apache.accumulo.core.trace.Trace"/>

--- a/pom.xml
+++ b/pom.xml
@@ -28,7 +28,7 @@
   <name>Apache Accumulo Testing</name>
   <description>Testing tools for Apache Accumulo</description>
   <properties>
-    <accumulo.version>2.0.0</accumulo.version>
+    <accumulo.version>2.1.0-SNAPSHOT</accumulo.version>
     <eclipseFormatterStyle>${project.basedir}/contrib/Eclipse-Accumulo-Codestyle.xml</eclipseFormatterStyle>
     <hadoop.version>3.2.0</hadoop.version>
     <maven.compiler.source>1.8</maven.compiler.source>

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/Module.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/Module.java
@@ -43,7 +43,7 @@ import javax.xml.validation.Schema;
 import javax.xml.validation.SchemaFactory;
 
 import org.apache.accumulo.core.client.security.tokens.PasswordToken;
-import org.apache.accumulo.core.util.SimpleThreadPool;
+import org.apache.accumulo.core.util.threads.ThreadPools;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.w3c.dom.Document;
@@ -224,7 +224,7 @@ public class Module extends Node {
       fixture.setUp(state, env);
     }
 
-    ExecutorService service = new SimpleThreadPool(1, "RandomWalk Runner");
+    ExecutorService service = ThreadPools.createFixedThreadPool(1, "RandomWalk Runner", false);
 
     try {
       Node initNode = getNode(initNodeId);

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Setup.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/bulk/Setup.java
@@ -26,7 +26,7 @@ import org.apache.accumulo.core.client.TableExistsException;
 import org.apache.accumulo.core.client.admin.TableOperations;
 import org.apache.accumulo.core.iterators.LongCombiner;
 import org.apache.accumulo.core.iterators.user.SummingCombiner;
-import org.apache.accumulo.core.util.SimpleThreadPool;
+import org.apache.accumulo.core.util.threads.ThreadPools;
 import org.apache.accumulo.testing.randomwalk.RandWalkEnv;
 import org.apache.accumulo.testing.randomwalk.State;
 import org.apache.accumulo.testing.randomwalk.Test;
@@ -60,8 +60,8 @@ public class Setup extends Test {
     state.set("fs", FileSystem.get(env.getHadoopConfiguration()));
     state.set("bulkImportSuccess", "true");
     BulkPlusOne.counter.set(0l);
-
-    ThreadPoolExecutor e = new SimpleThreadPool(MAX_POOL_SIZE, "bulkImportPool");
+    ThreadPoolExecutor e = ThreadPools.createFixedThreadPool(MAX_POOL_SIZE, "bulkImportPool",
+        false);
     state.set("pool", e);
   }
 

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Config.java
@@ -84,14 +84,14 @@ public class Config extends Test {
 			s(Property.TSERV_WAL_BLOCKSIZE, 1024 * 1024,
 					1024 * 1024 * 1024 * 10L),
 			s(Property.TSERV_WORKQ_THREADS, 1, 10),
-			s(Property.MASTER_BULK_THREADPOOL_SIZE, 1, 10),
-			s(Property.MASTER_BULK_RETRIES, 1, 10),
-			s(Property.MASTER_BULK_TIMEOUT, 10, 600),
-			s(Property.MASTER_FATE_THREADPOOL_SIZE, 1, 100),
-			s(Property.MASTER_RECOVERY_DELAY, 0, 100),
-			s(Property.MASTER_LEASE_RECOVERY_WAITING_PERIOD, 0, 10),
-			s(Property.MASTER_THREADCHECK, 100, 10000),
-			s(Property.MASTER_MINTHREADS, 1, 200),};
+			s(Property.MANAGER_BULK_THREADPOOL_SIZE, 1, 10),
+			s(Property.MANAGER_BULK_RETRIES, 1, 10),
+			s(Property.MANAGER_BULK_TIMEOUT, 10, 600),
+			s(Property.MANAGER_FATE_THREADPOOL_SIZE, 1, 100),
+			s(Property.MANAGER_RECOVERY_DELAY, 0, 100),
+			s(Property.MANAGER_LEASE_RECOVERY_WAITING_PERIOD, 0, 10),
+			s(Property.MANAGER_THREADCHECK, 100, 10000),
+			s(Property.MANAGER_MINTHREADS, 1, 200),};
 
 	Setting[] tableSettings = {
 			s(Property.TABLE_MAJC_RATIO, 1, 10),

--- a/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Replication.java
+++ b/src/main/java/org/apache/accumulo/testing/randomwalk/concurrent/Replication.java
@@ -17,7 +17,7 @@
 package org.apache.accumulo.testing.randomwalk.concurrent;
 
 import static com.google.common.util.concurrent.Uninterruptibles.sleepUninterruptibly;
-import static org.apache.accumulo.core.conf.Property.MASTER_REPLICATION_SCAN_INTERVAL;
+import static org.apache.accumulo.core.conf.Property.MANAGER_REPLICATION_SCAN_INTERVAL;
 import static org.apache.accumulo.core.conf.Property.REPLICATION_NAME;
 import static org.apache.accumulo.core.conf.Property.REPLICATION_PEERS;
 import static org.apache.accumulo.core.conf.Property.REPLICATION_PEER_PASSWORD;
@@ -74,7 +74,7 @@ public class Replication extends Test {
     iOps.setProperty(REPLICATION_PEER_USER.getKey() + instName, env.getAccumuloUserName());
     iOps.setProperty(REPLICATION_PEER_PASSWORD.getKey() + instName, env.getAccumuloPassword());
     // Tweak some replication parameters to make the replication go faster
-    iOps.setProperty(MASTER_REPLICATION_SCAN_INTERVAL.getKey(), "1s");
+    iOps.setProperty(MANAGER_REPLICATION_SCAN_INTERVAL.getKey(), "1s");
     iOps.setProperty(REPLICATION_WORK_ASSIGNMENT_SLEEP.getKey(), "1s");
     iOps.setProperty(REPLICATION_WORK_PROCESSOR_DELAY.getKey(), "1s");
     iOps.setProperty(REPLICATION_WORK_PROCESSOR_PERIOD.getKey(), "1s");

